composer/trainer/trainer.py

```diff
@@ -1186,7 +1197,7 @@ def fit(
             eval_interval (int | str | Time | (State, Event) -> bool, optional): See :class:`.Trainer`.
             grad_accum (int | str, optional): See :class:`.Trainer`.
             precision (Precision | str, optional): See :class:`.Trainer`.
-            grad_clip_norm (float, optional): See :class:`.Trainer`.
+            grad_clip_norm (float, optional): See :class:`.Trainer`. Deprecated. Please use composer.algorithms.GradientClipping
```

Comment:
Can you remove this docstring line? The arg was removed in the singature above.

Reply:
done here: [715c203](https://github.com/mosaicml/composer/pull/1099/commits/715c20344830a1f5a31b8a2962186ac7103971ab)